src/main/java/com/zepben/cimbend/database/sqlite/tables/iec61970/base/wires/TablePowerTransformers.kt

```diff
@@ -15,6 +15,7 @@ import com.zepben.cimbend.database.sqlite.tables.iec61970.base.core.TableConduct
 class TablePowerTransformers : TableConductingEquipment() {
 
     val VECTOR_GROUP = Column(++columnIndex, "vector_group", "TEXT", NOT_NULL)
+    val TRANSFORMER_UTILISATION = Column(++columnIndex, "transformer_utilisation", "NUMBER", NOT_NULL)
```

Comment:
Change to a database table without changing the database changelog number and creating an upgrade script